server/executor/tracepollerworker/fetcher_worker.go

```diff
@@ -47,6 +49,11 @@ func (w *traceFetcherWorker) SetOutputQueue(queue pipeline.Enqueuer[executor.Job
 }
 
 func (w *traceFetcherWorker) ProcessItem(ctx context.Context, job executor.Job) {
+	if !w.enabled {
+		w.outputQueue.Enqueue(ctx, job)
```

Comment:
shouldn't this be just a skip? if we have an external queue processor that also listens for this step and then passes it to the next, the item will be duplicated in the queue

Reply:
I agree with you, changing it now.